web/src/main/java/se/inera/intyg/intygsadmin/web/auth/IntygsadminUser.java

```diff
@@ -44,4 +50,25 @@ public IntygsadminUser(UserEntity userEntity, AuthenticationMethod authenticatio
         this.name = userEntity.getName();
     }
 
+    @Override
```

Comment:
We can use Lombok annotations for providing implementation of equals and hashcode